integration-tests/elasticsearch-rest-client/src/test/java/io/quarkus/it/elasticsearch/FruitResourceTest.java

```diff
@@ -2,56 +2,88 @@
 
 import static io.restassured.RestAssured.get;
 import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
 
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
 import io.restassured.common.mapper.TypeRef;
+import io.restassured.parsing.Parser;
 
 @QuarkusTest
 public class FruitResourceTest {
     private static final TypeRef<List<Fruit>> LIST_OF_FRUIT_TYPE_REF = new TypeRef<>() {
     };
 
     @Test
-    public void testEndpoint() throws InterruptedException {
+    public void testEndpoint() {
+        RestAssured.defaultParser = Parser.JSON;
```

Comment:
I wonder if we are just not missing a content type header in some request/response ?

Reply:
I didn't quite understand the meaning of the question. Do you mean that in this context this line makes no sense?

Reply:
😃 what I was trying to say ... I'd expect that RestAssured figures out the parser on its own, and we don't need to specify one explicitly. But if it fails somewhere with an exception that it cannot parse the body, then maybe we are just missing a `@Produces` annotation somewhere in our rest endpoint...

Reply:
Well, year, got your point finally. Thank you for clarification :)

Reply:
> 😃 what I was trying to say ... I'd expect that RestAssured figures out the parser on its own, and we don't need to specify one explicitly. But if it fails somewhere with an exception that it cannot parse the body, then maybe we are just missing a `@Produces` annotation somewhere in our rest endpoint...

I've made all the changes.🌚